src/clarity-angular/datagrid/providers/sort.spec.ts

```diff
@@ -3,37 +3,39 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import {Sort} from "./sort";
-import {Comparator} from "../interfaces/comparator";
+import { Sort } from "./sort";
+import { Comparator } from "../interfaces/comparator";
 
-export default function(): void {
-    describe("Sort provider", function() {
-        beforeEach(function() {
+export default function (): void {
+    describe("Sort provider", function () {
+        beforeEach(function () {
             this.sortInstance = new Sort();
             this.comparator = new TestComparator();
         });
 
-        it("compares according to the current comparator", function() {
+        it("compares according to the current comparator", function () {
```

Comment:
Do we have any tests for the default setting or if a comparator comes through as null or undefined?

What happens in those cases?